### PR TITLE
Double keybinding

### PR DESCRIPTION
**Pull request by Dimava**
Allow players to use two keybinds at once

Allows to make a CUT keybind by making secondary key for both COPY and DELETE at once

Notes:
 - implementation may be changet to array of keys, allowing even more keybinds and unbinding LMB/RMB from default actions
 - settings styling may be somewhat tweaked

### CODE REVIEW

File: src/js/game/hud/parts/mass_selector.js [hunk 1]
```diff
@@ -116,7 +116,11 @@ export class HUDMassSelector extends BaseHUDPart {
                 return;
             }
             this.root.hud.signals.buildingsSelectedForCopy.dispatch(Array.from(this.selectedUids));
-            this.selectedUids = new Set();
+            if (
+                !this.root.keyMapper.getBinding(KEYMAPPINGS.massSelect.confirmMassDelete).isCurrentlyPressed()
+            ) {
+                this.selectedUids = new Set();
+            }
```
Comment (tobspr) [inline src/js/game/hud/parts/mass_selector.js:123]: What exactly is this for?

**Comment by Dimava** [comment]
so the only thing left is styling

File: src/js/game/hud/parts/mass_selector.js (hunk 1 above)
Comment (Dimava) [reply src/js/game/hud/parts/mass_selector.js:123]: to allow CUT command

**Comment by tobspr** [comment]
Closing this for now since it hasn't been requested by anybody so far

**Pull request closed without merging**